assets/scss/components/_service-info.scss

```diff
@@ -195,3 +195,48 @@ Styleguide Service Info.Logged In
 .last-login__more-details {
   margin: em(5) 0 0 0;
 }
+
+/*
+Floated items
+
+Service Info with `.service-info__floated-items` allows for content to be floated on the left and right sides with adequate margins. On mobile breakpoints content is displayed block
+
+Markup:
+<div class="service-info ">
+  <div class="service-info__floated-items">
+    <div class="service-info__left-item">
+      <span>Left Side</span>
+    </div>
+    <div class="service-info__right-item">
+      <span>Right Side</span>
+    </div>
+  </div>
+</div>
+
+Styleguide Service Info.Left and Right Items
+*/
+.service-info__floated-items {
```

Comment:
Or if we're moving the margin to padding later just add the proposed `-left` and -`right` classes in the comment below and then we're not adding unnecessary extras to the markup that we'll just have to remove later